providers/src/airflow/providers/celery/executors/celery_executor.py

```diff
@@ -433,31 +434,34 @@ def try_adopt_task_instances(self, tis: Sequence[TaskInstance]) -> Sequence[Task
 
         return not_adopted_tis
 
+    @deprecated(
+        reason="Replaced by function `revoke_task`. Upgrade airflow core to make this go away.",
```

Comment:
hmm, I wonder if we should instead not deprecate this, but leave a comment that it can be removed when the min airflow version for providers is 2.11?

Reply:
Why not deprecate?  This warning would only appear if user upgrades celery but not airflow

Reply:
Yeah. It doesn't really matter. Just feels a bit cleaner to me.

We should at least state what version to upgrade to.

Reply:
we don't know what version this is gonna be released in, alas

Reply:
it would also only show up in scheduler logs, and only when stuck quueued tasks processed